KomaMRIBase/src/datatypes/Sequence.jl

```diff
@@ -724,3 +724,12 @@ get_eddy_currents(seq::Sequence; Δt=1, λ=80e-3) = begin
 	#Final
 	M2, M2_adc
 end
+
+"""
+It is the same as defined in MRIFiles without the need to import MRIFiles
+"""
+struct Limit
+    minimum::Int
+    maximum::Int
+    center::Int
+end
```

Comment:
Remove this.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/commit/822fea1165c72a77178b98278ecf9ce4cfa55f96